teachers_digital_platform/css/layout/sidebar-1-2.less

```diff
@@ -0,0 +1,61 @@
+.l-sidebar-1-2 {
```

Comment:
I'm having a bit of difficulty getting this up and running for testing locally, but I'm wondering what the styles in this file are doing. At a glance, this looks like the CSS output of some of our typical cf-layout Less; would you be able to leverage the stuff already provided in cf-layout, or do you need to do something different?

Reply:
I couldn't get the cf-layout to output a layout with a 33% sidebar and a 66% main column.  I tried adding a layout-1-2.html that I could extend things from but it didn't work for me.  Hence this sidebar-1-2.less sheet that builds the layout I want.

Reply:
I'm off an vacation next week.  Dan's back on Thursday and can probably refactor the layout.